### PR TITLE
Fix bug: the websocket connection timeout setting doesn't take effect

### PR DESCRIPTION
**Pull request by XiaoJiangWang**
**What type of PR is this?**

**What this PR does / why we need it**:
Set the timeout period before sending and reading data from the long connection

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by daixiang0** [comment]
Is it better to set timeout when create connection?

**Comment by XiaoJiangWang** [comment]
> Is it better to set timeout when create connection?

you can find the example in gorilla/websocket.
https://github.com/gorilla/websocket/blob/master/examples/chat/client.go#L91

The write deadline must be setted before every sending process, because the timeout time stamp is calculated from the current time.

**Comment by daixiang0** [comment]
lgtm

File: edge/pkg/edgehub/clients/wsclient/websocket.go [hunk 1]
```diff
@@ -105,13 +105,22 @@ func (wsc *WebSocketClient) UnInit() {
...
+		return err
+	}
 	return wsc.connection.WriteMessageAsync(&message)
 }
 
 //Receive reads the binary message through the connection
 func (wsc *WebSocketClient) Receive() (model.Message, error) {
+	var err error
 	message := model.Message{}
-	err := wsc.connection.ReadMessage(&message)
```
Comment (fisherxu) [inline edge/pkg/edgehub/clients/wsclient/websocket.go:114]: Looks good, but seems `ReadMessage` hadn't use this Deadline in  Underlying `WSConnection`?

Comment (fisherxu) [reply edge/pkg/edgehub/clients/wsclient/websocket.go:114]: Writing use it here: https://github.com/kubeedge/kubeedge/blob/master/staging/src/github.com/kubeedge/viaduct/pkg/conn/ws.go#L179

Comment (XiaoJiangWang) [reply edge/pkg/edgehub/clients/wsclient/websocket.go:114]: > Looks good, but seems `ReadMessage` hadn't used this Deadline in Underlying `WSConnection`?

Yes. That's my mistake. `ReadMessage` gets messages from messageFifo in `WSConnection`, the read deadline setting also does't take effect in this method.
But CloudHub won't regularly send messages to the edge to make sure the websocket connection is alive. So is it necessary to set a timeout when read message?

Comment (XiaoJiangWang) [reply edge/pkg/edgehub/clients/wsclient/websocket.go]: delete the code for setting read message deadline

Comment (fisherxu) [reply edge/pkg/edgehub/clients/wsclient/websocket.go]: ok, I'm also not sure, maybe we don't need set read deadline?

**Pull request merged**